lib/genMixin.js

```diff
@@ -6,18 +7,11 @@ const readFile = util.promisify(fs.readFile)
 const writeFile = util.promisify(fs.writeFile)
 
 async function generateMixin(styles, lang, outputFolder) {
-    let templateFile
-    let ext
-    let name
-    if (lang == "less") {
-        templateFile = `/../lib/template.less.hb`
-        ext = "less"
-        name = "LESS"
-    } else { // if lang == "sass"
-        templateFile = `/../lib/template.scss.hb`
-        ext = "scss"
-        name = "SASS"
-    }
+    const styleInformation = getStyleInformation(lang)
+    let templateFile = styleInformation.templateFile
+    let ext = styleInformation.ext
+    let name = styleInformation.name
```

Comment:
These could now be `const`, or if you don't want them here you could use `styleInformation.XX` directly where you are using these.

Reply:
Oh, sure. I removed them...